test/features/BoolTest.java

```diff
@@ -692,15 +696,15 @@ public void testBoolSafeLoadXML() {
         XMLFileWriter.toXML("testOutput/BoolAutomaton.xml",new Automaton[]{aut});
         Automaton newAut = XMLParser.parse("testOutput/boolAutomaton.xml",false)[0];
         XMLFileWriter.toXML("testOutput/BoolAutomatonNew.xml",new Automaton[]{newAut});
-        System.out.println("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!");
+        Log.debug("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!");
```

Comment:
Probably :smile: but my guess would be that they were used as a separator for the comented refinement checks. As they are commented I think it would be OK to delete the debug logs.

Reply:
Yes, just delete these ancient debug print lines.